runtime/oti/VMHelpers.hpp

```diff
@@ -2052,16 +2052,22 @@ class VM_VMHelpers
 	}
 
 #if JAVA_SPEC_VERSION >= 19
+	static VMINLINE ContinuationState volatile *
+	getContinuationStateAddress(J9VMThread *vmThread , j9object_t object)
+	{
+		return (ContinuationState volatile *) ((uintptr_t) object + J9VMJDKINTERNALVMCONTINUATION_STATE_OFFSET(vmThread));
+	}
+
 	static VMINLINE bool
 	isStarted(ContinuationState continuationState)
 	{
 		return J9_ARE_ALL_BITS_SET(continuationState, J9_GC_CONTINUATION_STATE_STARTED);
 	}
 
 	static VMINLINE void
-	setContinuationStarted(J9VMContinuation *continuation)
+	setContinuationStarted(ContinuationState *continuationStatePtr)
```

Comment:
I believe that any of these methods that pass pointer will have to be (compile will make you do it) with volatile